PHPUnit/Util/ErrorHandler.php

```diff
@@ -99,7 +99,8 @@ public static function handleError($errno, $errstr, $errfile, $errline)
             }
         }
 
-        if ($errno == E_NOTICE || $errno == E_USER_NOTICE || $errno == E_STRICT) {
+        if ($errno == E_NOTICE || $errno == E_USER_NOTICE
+            || (PHP_VERSION_ID < 70400 && $errno == E_STRICT)) {
```

Comment:
```suggestion
            || (PHP_VERSION_ID < 80400 && $errno == E_STRICT)) {
```

Shouldn't the deprecation start with 8.4? not 7.4?

Reply:
@glensc According to this, some functions already emit deprecation notice starting 7.4:
https://php.watch/versions/8.4/E_STRICT-deprecated

Reply:
from the linked article:
>  It is safe to assume that any PHP applications that run on PHP 8.0 and later will never encounter E_STRICT notices

but still the correct check would be `<= 70400` or `< 80000` instead of `< 70400` @marcing